src/bitmessageqt/languagebox.py

```diff
@@ -8,24 +10,27 @@
 
 
 class LanguageBox(QtGui.QComboBox):
+    """LanguageBox class for QT UI"""
     languageName = {
         "system": "System Settings", "eo": "Esperanto",
         "en_pirate": "Pirate English"
     }
 
     def __init__(self, parent=None):
+        """Initialization"""
         super(QtGui.QComboBox, self).__init__(parent)
         self.populate()
 
     def populate(self):
+        """Populate comboBox based on languageName"""
         self.clear()
         localesPath = os.path.join(paths.codePath(), 'translations')
         self.addItem(QtGui.QApplication.translate(
             "settingsDialog", "System Settings", "system"), "system")
         self.setCurrentIndex(0)
         self.setInsertPolicy(QtGui.QComboBox.InsertAlphabetically)
         for translationFile in sorted(
-            glob.glob(os.path.join(localesPath, "bitmessage_*.qm"))
+                glob.glob(os.path.join(localesPath, "bitmessage_*.qm"))
```

Comment:
not necessary

Reply:
@g1itch pylint complains for it 

C: 33, 0: Wrong hanging indentation before block (add 4 spaces).
            glob.glob(os.path.join(localesPath, "bitmessage_*.qm"))
            ^   | (bad-continuation)

Reply:
It's a bug: https://github.com/psf/black/issues/48

Reply:
I tired to point all of you to the same bugs and PEP8 articles...

Reply:
@g1itch so for now I should undo this ?

Reply:
undo change and override pylint

Reply:
Done.